content/english/Kusto-KC7/activity-0.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "Activity 0: Login & Setup of the Cyber Environment"
+description: ""
+draft: false
+weight: 4
+---
+
+{{< youtube VL7fJr54CG0 >}}
+
+# Getting Started
+
+Okay Cyber Defenders! The first thing you need to do is set up the environment in which you will be hunting hackers. In order to do that, open the links below and follow the directions on first setting up Azure Data Explorer (ADX) and then logging into the Scoreboard session.
+- Azure Data Explorer : https://dataexplorer.azure.com/ 
```

Comment:
This might be confusing - we ask students to click on this when we go into the setup steps below in more detail .It might make more sense to just remove these and just let the setup steps below guide the students through these links.

Reply:
ok i removed it